server/app/views/admin/programs/ProgramIndexViewV2.java

```diff
@@ -141,7 +141,7 @@ private Modal renderDemographicsCsvModal() {
                                     .withClasses(Styles.ML_3, Styles.INLINE_FLEX),
                                 FieldWithLabel.date()
                                     .setFieldName("untilDate")
-                                    .setLabelText("To:")
```

Comment:
Note to the reviewer: I forgot to rename To -> Until when originally adding time filters to the demography export.